gma/src/include/firebase/gma/internal/README.md

```diff
@@ -0,0 +1,6 @@
+# Firebase C++ Open Source Development
+
+**NOTE: For internal use only.**
+
+This folder contains experimental header files that are internal and experimental.
+Users shouldn't expect any stability guarantees for code in this folder, and any public usage is discouraged and  will not be supported.
```

Comment:
Can we change "will not be supported" to "is not supported"?

Reply:
Ack.